style.css

```diff
@@ -2512,7 +2561,108 @@ body {
     height: 2788px;
     left: 0;
     top: 718px;
-    background-color: #e5e5e5;
+    background-color: var(--body-background-colour);
+  }
+
+  .portfolio-popup {
+    display: none;
+    position: absolute;
+    justify-content: center;
+    top: 40px;
+    width: 375px;
+    margin-left: 20px;
+  }
```

Comment:
- [ ] Hi @gaks1, great effort working on the popup modal of your portfolio project, currently i notice some issues that required your attention in the popup for both the mobile and desktop version. 👏 👏 

   - The nav bar in the desktop version scroll over the popup modal anytime we scroll the popup card,
   - On the mobile view, the hamburger mobile menu displayed when clicked on despite the popup modal is displayed. This is strange functionality kindly fix it.
   - I would kindly suggest you reduce the text on the popup card so that the card can fit in to the screen when popped and there be any reason to scroll up and down, and this will give a better user experience.

Your mobile view 👇 

![iss](https://user-images.githubusercontent.com/78634841/216276447-963312c3-260d-40d5-9b61-4753888351a2.png)